app/src/main/res/layout/synced_history_nav_item.xml

```diff
@@ -18,7 +18,7 @@
         android:id="@+id/icon"
         android:layout_width="@dimen/history_favicon_width_height"
         android:layout_height="@dimen/history_favicon_width_height"
```

Comment:
It's possible that we can convert both this and recently_closed_nav_item to use 24x24 instead of 40x40 width and height and drop the padding if you want to simplify this. There's no image switching or tap handling for both of these ImageViews so that should simplify matters if anyone ever wonders why the margin start is 8dp instead of 16dp and doesn't realize that you're accounting for the 40x40 size.

Reply:
done